### PR TITLE
#303 Add donations list to the admin panel

### PR DESCRIPTION
**Pull request by Sontsevolosa**
Resolves [github issue](https://github.com/pivorakmeetup/pivorak-web-app/issues/303)

### Description

- Add route for donations list to the routes.rb file
- Add link to donations list to the navigation helper
- Add link to donations list to the footer of admin panel
- Create Donations controller in the Admin scope
- Get list of all donations in 'donations' helper method in the Admin/Donations controller
- Add table with all donations to the view for 'index' action
- Add localization of 'Donator' word to the en.yml file
- Add pagination for donations
- Add breadcrumbs to the view for 'index' action
- Add test for checking list of donations on the page

### How to test instructions

rspec/spec/features/admin/donations

### CODE REVIEW

**Comment by Sontsevolosa** [comment]
@denysmedynskyi please, review my PR 🙂

File: app/views/admin/donations/index.slim [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  thead
+    tr
+      th.center.aligned = t 'donations.donator'
+      th.center.aligned = t 'goals.amount'
+      th.center.aligned = t 'words.date'
+
+  tbody
+    - donations.each do |donation|
+      tr
+        - if donation.user
```
Comment (denys-medynskyi) [inline app/views/admin/donations/index.slim]: You can do `user&.full_name || '-'` instead of `if`

File: app/views/admin/donations/index.slim [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+          td.center.aligned = "#{donation.user.first_name} #{donation.user.last_name}"
+        - else
+          td.center.aligned = "-"
+        td.center.aligned = donation.amount.to_f
+        td.center.aligned = donation.created_at.strftime("%d/%m/%Y")
```
Comment (denys-medynskyi) [inline app/views/admin/donations/index.slim]: Extract `%x%x%x` to en.yml. Check this [article](http://www.stackednotion.com/blog/2015/01/03/formatting-dates-and-times-in-rails-with-i18n-localize/)

**Review by denys-medynskyi — CHANGES_REQUESTED** [review]
Good job!
However, I have a few suggestions. In the code and one more in UI - https://cl.ly/2Y0l223L1N0P - to add `total` on the top of the page

**Comment by Sontsevolosa** [comment]
@denysmedynskyi please, review my changes)

**Review by denys-medynskyi — APPROVED** [review]
Everything is great!
However, there is a thing about the PR description: how to test means - how to MANUALLY test your PR.

**Pull request merged**